.github/workflows/lint.yml

```diff
@@ -0,0 +1,28 @@
+name: "Lint"
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ "*" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1.152.0  
+        with:
+          ruby-version: '3.2.2'
+          bundler-cache: true  # runs 'bundle install' and caches installed gems automatically
+    
+      # - name: Add vendor directory to path 
+      #   run:  echo "$GITHUB_WORKSPACE/vendor/bundle/ruby/3.2.0/bin" >> $GITHUB_PATH 
+      
+      - name: Security audit dependencies
+        run: bundle exec bundler-audit check --update 
+
+      - name: Security audit application code
+        run: bundle exec brakeman -q -w2
+
+      - name: Lint Ruby files
+        run: bundle exec rubocop --parallel
```

Comment:
We should add `rubocop` directly into the repo. `stardardrb` is a wrapper around rubocop with an unconfigurable set of lint conventions so we don't have to bikeshed on lint rules.

So, `bundle exec standardrb` (I think) should be what's evaluated here, and you can drop the two added rubocop dependencies. So no Gemfile/Gemfile.lock diffs whatsoever.

Reply:
Please see https://github.com/rubyforgood/stocks-in-the-future/pull/26/commits/23cfe8e13da689d7bf07fe2fbb2309dccf7188a8